package.json

```diff
@@ -14,8 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids",
     "gen-api-docs-all": "docusaurus gen-api-docs all --all-versions",
     "gen-api-docs-clean": "docusaurus clean-api-docs all",
-    "check-vendored-yaml": "tsx scripts/check-vendored-yaml.ts",
-    "update-vendored-yaml": "tsx scripts/update-vendored-yaml.ts"
+    "check-vendored-yaml": "tsx src/openapi/check-vendored-yaml.ts",
+    "update-vendored-yaml": "tsx src/openapi/update-vendored-yaml.ts"
```

Comment:
any advantage to moving these?

Reply:
Yep (and good question).  The advantage is that now all the items relating to OpenAPI are under the same location (`src/openapi`).  

It also aligns with https://github.com/virtru-corp/dsp-docs/pull/93